yt/frontends/stream/tests/test_stream_species.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
+
+from yt.loaders import load_uniform_grid
+from yt.testing import assert_allclose_units
+
+
+def test_stream_species():
+    arr = np.random.uniform(size=(32, 32, 32))
+
+    data = {
+        "density": (arr, "g/cm**3"),
+        "H_p0_fraction": (0.37 * np.ones(arr.shape), "dimensionless"),
```

Comment:
Again, `np.ones_like(arr)` is slightly preferred but not a huge deal.

Reply:
Done